site/reference/pkg/update/README.md

```diff
@@ -67,33 +43,47 @@ VERSION:
 
 ```
 --strategy:
-  Controls how changes to the local package are handled.  Defaults to fast-forward.
-
-    * resource-merge: perform a structural comparison of the original /
-      updated Resources, and merge the changes into the local package.
-    * fast-forward: fail without updating if the local package was modified
+  Defines which strategy should be used to update the package. This will change
+  the update strategy for the current kpt package for future updates. If a
+  strategy is not provided, the strategy specified in the package Kptfile will
+  be used.
+
+    * resource-merge: Perform a structural comparison of the original /
+      updated resources, and merge the changes into the local package.
+    * fast-forward: Fail without updating if the local package was modified
       since it was fetched.
-    * alpha-git-patch: use 'git format-patch' and 'git am' to apply a
-      patch of the changes between the source version and destination
-      version.
-    * force-delete-replace: WIPE ALL LOCAL CHANGES TO THE PACKAGE.
-      DELETE the local package at local_pkg_dir/ and replace it
-      with the remote version.
-
--r, --repo:
-  Git repo url for updating contents.  Defaults to the repo the package
-  was fetched from.
-
---dry-run
-  Print the 'alpha-git-patch' strategy patch rather than merging it.
+    * force-delete-replace: Wipe all the local changes to the package and replace
+      it with the remote version.
 ```
 
 #### Env Vars
 
 ```
 KPT_CACHE_DIR:
-  Controls where to cache remote packages when fetching them to update
-  local packages.
-  Defaults to ~/.kpt/repos/
+  Controls where to cache remote packages when fetching them.
+  Defaults to <HOME>/.kpt/repos/
+  On macOS and Linux <HOME> is determined by the $HOME env variable, while on
+  Windows it is given by the %USERPROFILE% env variable.
+```
+<!--mdtogo-->
+
+### Examples
+<!--mdtogo:Examples-->
+```shell
+# Update package in the current directory.
+git add . && git commit -m 'some message'
+kpt pkg update
+```
+
+```shell
+# Update my-package-dir/ to match the v1.3 branch or tag.
+git add . && git commit -m 'some message'
+kpt pkg update my-package-dir/@v1.3
```

Comment:
It should be possible to do this:

```
cd wordpress
$ kpt pkg update @v0.2
```

In other words, the usage is

```
update [PKG_PATH][@VERSION] [flags]
```

@phanimarupaka to check for consistency

Reply:
Yes. That works. Verified it.

Reply:
Updated the usage to reflect this,